### ISSUE TITLE
Limit part scrapping to Engineers

### ISSUE THREAD

**Issue by DanStaal** [BobPalmer/MKS#1086]
The thought is that it takes someone who knows what they are doing to take apart ships into usable parts, instead of just turning them into scrap. (Since recyclables are a thing, others could maybe turn ships into recyclables.)  Anyone can activate a demolition block though - that just takes pressing a button.  (And an engineer, either back at the KSC or on EVA via KIS is needed to place the block correctly - the latter being KIS's default behavior.)

Subclassess to, of course.  Quartermaster being an odd case; a good quartermaster should be able to re-purpose things, and could probably do this, despite it being Pilot subclass.  (Miner, Technician, and Mechanic should all be able to fairly obviously.)

**Comment by dgbinger** [comment]
I'd be excited for the "non-engineers scrap to recyclables" option, or for a settings option maybe.

I was very excited to have my engineers no longer the only kerbals who could keep the base clean, though, so I'd be sad to see a full-scale rollback.

**Comment by DanStaal** [comment]
Note that even in my original suggestion, a non-engineer can keep the base clean, assuming you've done some planning during ship design: Just liberally attach demolition blocks to everything you'll want to clean up, and anyone can scrap them.

**Comment by dgbinger** [comment]
True, as long as you've planned ahead (and your plans haven't changed too much: if there are parts you weren't expecting to need but now want, for instance...) I'm, uh, not always that smart.

All other things being equal, it seems like non-engineers ought to be able to get rid of loose bits without needing pre-planted charges. Engineers can render a fuel tank or engine into usable materialkits (with their bare hands, apparently), so a pilot probably ought to be able to turn it into nothing/garbage, right?

I like the "recyclables" idea as well, just because I'm always on the fence about whether it's actually worth adding a recycler to my bases: I like the idea of re-using that stuff aesthetically, but it's a big part that only does one thing, and a refinery will often be more effective. Adding more sources for recyclables makes it a better bet.
